src/lib/numbers.js

```diff
@@ -0,0 +1,57 @@
+function add (a, b) {
```

Comment:
The formatting here is a bit out of whack, it would be a great idea to run the formatter across this and let it do the heavy lifting for you